File: festival-kotlin-web/src/main/java/cn/bdqfork/web/route/response/ResponseHandleStrategy.java
package cn.bdqfork.web.route.response;


import io.vertx.core.http.HttpServerResponse;

/**
 * @author bdq
 * @since 2020/1/30
 */
public interface ResponseHandleStrategy {

    void handle(HttpServerResponse httpServerResponse, Object result) throws Exception;
}

File: festival-kotlin-web/src/main/kotlin/cn/bdqfork/kotlin/web/route/response/ResponseHandleStrategy.kt
package cn.bdqfork.kotlin.web.route.response

import io.vertx.ext.web.RoutingContext

/**
 * @author bdq
 * @since 2020/1/30
 */
interface ResponseHandleStrategy {
    @Throws(Exception::class)
    fun handle(routingContext: RoutingContext, result: Any?)
}